### ISSUE TITLE
Scheduling placeholder matches

### ISSUE THREAD

**Issue by Astraeus-** [Fabian-Sommer/HeroesLounge#12]
Add an option to the scheduling page to select in a placeholder option instead of scheduling your match on an unlikely to playing at time.
E.g: How teams current have to put in 4:00 AM on day x, they'd instead just press a button "placeholder".

It'd make it more obvious for newer teams to put in a placeholder and for our casters to not confuse placeholders with legitimate matches when they aren't scheduled for unlikely time slots.

**Comment by rogusdev** [comment]
What would the placeholder be for?  Would this be covered by letting teams reschedule matches themselves?  https://github.com/Fabian-Sommer/HeroesLounge/issues/61

**Comment by Astraeus-** [comment]
Not quite, but I'd like to deploy the two together. Teams need to submit a date within the first week of a match to prevent going inactive, however not all teams can agree on a date within this first week. They instead schedule for an unrealistic time, which can lead to confusion. Instead a placeholder option would alleviate this.

**Comment by rogusdev** [comment]
So say they push "placebolder" button, does this simply extend their time to schedule by a week?

**Comment by Astraeus-** [comment]
You'd have to check the matchmaking code, but personally I wouldn't update the deadline dates to achieve this functionality. Likely easier to add placeholder to the match model, put in a "random" date and let teams reschedule through #61 when that's complete and contact moderators until then as is the norm.

**Comment by rogusdev** [comment]
I don't follow.  What is the placeholder on the match model, and why pick a random date?  Why not simply encourage teams to pick some date and reschedule on their own?  Or pick the latest possible date and reschedule, etc.

**Comment by Astraeus-** [comment]
I was suggesting to add a field `placeholder` to indicate that a match does not have its definitive play date. I'd think this would also allow us to easily filter out these matches from the calendar. A date is required to be attached, because I believe the matchmaker checks for one to determine if a team is active each week. 

In regards to picking the date, there's been times where a team submits a placeholder time, which would be feasible to play at, baiting casters to sign up. It's also not entirely obvious picking the last day or time is how to put in a placeholder. 

Currently teams need to contact a moderator to reschedule their matches which #61 should fix.

**Comment by Fabian-Sommer** [comment]
I think you may confuse the issue at hand here a bit: Right now, Teams that do not schedule within a week are assumed to be inactive. Teams use placeholder times to prevent that from happening. What we really want is for teams not having to use this workaround, but schedule only when they know when the match will take place.

This means that we would like to have some other way for captains to confirm that they are still active, even when they cannot schedule within the first week. This would eliminate the need for placeholder dates in the first place.

All teams that do schedule within the first week should be active, as well as all teams whose captains push a button somewhere (the button can be on the teams management page or the match scheduling/reporting overview page or both). Teams who fulfill neither condition should be made inactive whenever matchmaking runs.

**Comment by eayildiz** [comment]
We can schedule matches in background but make it invisible to users so caster cannot sign up to it. Or we can change inactivity condition so that teams who checked placeholder won't be inactive but they don't have a scheduled match as well. I need to check code if both can be doable. But it sounds like second option is easier.
